cfgov/v1/auth_forms.py

```diff
@@ -122,4 +122,11 @@ def confirm_login_allowed(self, user):
             raise ValidationError("This account is temporarily locked; please try later or <a href='/admin/password_reset/' style='color:white;font-weight:bold'>reset your password</a>")
 
 
+class UserCreationForm(wagtailforms.UserCreationForm):
+    def save(self, commit=True):
+        user = super(UserCreationForm, self).save(commit=commit)
 
+        if commit:
+            send_password_reset_email(user.email)
```

Comment:
Do you want to open a ticket in the GHE platform repo to discuss this? I doubt we'll reach any conclusions for this PR, but we should definitely be logging things like that, and it'd be nice to establish a convention and document it (in @grapesmoker's awesome docs collection).